### PR TITLE
Replace team_abbr and model_abbr output columns with model_id

### LINKED ISSUE #2: Update round, team, and model parsing in model-output transform function
[2024-05-03: updated ticket based on recent convos/decisions that emerged from @lmullany's work to convert the archived FluSight hub to Hubverse format: [model-id = model name + team name](https://github.com/Infectious-Disease-Modeling-Hubs/hubDocs/issues/114) and [is round-id required to be in yyyy-mm-dd format](https://github.com/Infectious-Disease-Modeling-Hubs/hubValidations/issues/81).]

**Background**
The first iteration of the `hubverse-transform` function copied logic from the Hubverse validation function to get round_id, model name, and team name.

However, the ongoing work to convert archived hub data to Hubverse format resulted in some good conversations about these data, so `hubverse-transform` needs to be updated to reflect the resulting decisions.

**Required work**
Update the [`hubverse_transforms` module](https://github.com/Infectious-Disease-Modeling-Hubs/hubverse-transform/tree/main/src/hubverse_transform) to add 2 columns to incoming model-output files instead of 3 (model_id and round_id instead of model_name, team_name, and round_id).

**Definition of done**
- [x] `hubverse_transforms` adds two columns to incoming model-output files submitted via cloud-based hubs: `model_id` and `round_id` 

**Note:** Until we [make a decision otherwise](https://github.com/Infectious-Disease-Modeling-Hubs/hubValidations/issues/81), this code will continue to assume that hub round_ids are in YYYY-MM-DD format). Our first big test of the transforms will be on the incoming FluSight archives, and [those model-output files use YYYY-MM-DD](https://github.com/lmullany/flusight1_hub/tree/main/model-output), so we're good for now.

### PR DESCRIPTION
**Pull request by bsweger**
Resolves #2 

Given @lmullany's findings when converting the archived FluSight repo into hubverse format, we made a decision to simplify how we're parsing model-output file names.

TL;DR: instead of creating separate model and team abbr columns in the transformed parquet file, we're going to create a single `model_id` column (see the linked issue for more details).

This PR also sneaks in a minor change to logging strings based on feedback from @matthewcornell

### CODE REVIEW

File: src/hubverse_transform/model_output.py [hunk 1]
```diff
@@ -70,23 +69,26 @@ def from_s3(cls, bucket_name: str, s3_key: str, origin_prefix: str = "raw") -> "
...
+        round_id_match = re.match(r"^\d{4}-\d{2}-\d{2}", file_name)
+        if not round_id_match:
+            raise ValueError(f"Unable to get YYYY-MM-DD round_id from file name {file_name}.")
+        round_id = round_id_match.group(0)
+
+        # model_id is anything that comes after round_id in the filename
+        model_id_split = re.split(rf"{round_id}[-_]*", file_name)
+        if not model_id_split or len(model_id_split) <= 1 or not model_id_split[-1]:
+            raise ValueError(f"Unable to get model_id from file name {file_name}.")
+        model_id = "".join(model_id_split[-1].split())
```
Comment (lmullany) [inline src/hubverse_transform/model_output.py:87]: this removes any spaces in whatever is remaining. Is that your intent? So `2022-12-31-my team-my model.parquet` would return `"myteam-mymodel"` for `model_id`

**Review by lmullany — APPROVED** [review]
look good to me.

File: src/hubverse_transform/model_output.py (hunk 1 above)
Comment (bsweger) [reply src/hubverse_transform/model_output.py:87]: Good call-out. It was my intent at the time I made the pull request, but I have another in the queue (started after Wednesday's dev meeting) that retains the spaces and adds some corresponding test cases.
